vms/platformvm/block/executor/options.go

```diff
@@ -175,10 +175,8 @@ func (o *options) prefersCommit(tx *txs.Tx) (bool, error) {
 		expectedUptimePercentage = float64(transformSubnet.UptimeRequirement) / reward.PercentDenominator
 	}
```

Comment:
I think we can just use the primary network uptime requirement here now as well.

Reply:
Thinking about this again - I think that change would make more sense in a different PR.